report/implementations.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "docker-image": "gregsdennis/json-everything",
+        "command": "dotnet json-everything.dll",
```

Comment:
This should probably be an array to avoid the off-chance of needing shell quoting.

Reply:
I don't understand.

Reply:
Agreed - wouldn't special characters already be escaped? (and json should only require escaping `"` anyway, I think.)

Reply:
(I deleted my previous "I agree that this doesn't make sense" comment, after thinking about it harder) :)

If a command is provided as a single string, then the implication is it's passed to the shell, which may parse some characters unless they're escaped; if the command is passed as an array of words, then we can bypass the shell entirely and just hand it to `exec` (or equivalent).

e.g. perl's `exec` and `system` functions pass the command through the shell if one argument is provided, but skips that step if multiple arguments are provided, with the presumption that tokenization has already been done (and without having to fuss with escaping characters that the shell might do something with).  https://perldoc.perl.org/functions/exec

Reply:
Okay. Again. I'm learning as I go here, so feel free to slow am example or add a commit.  I don't know how to translate a string array into a bash command.

Reply:
That's what I was referring to yeah -- and no worries at all! I can leave some commits showing what I mean -- it's true, shell scripting indeed doesn't make some of this trivial.